app/mapping/reads/TrusteeIndividual.scala

```diff
@@ -82,7 +75,7 @@ object TrusteeIndividual extends TrusteeReads[TrusteeIndividual] {
         passportOrIdCardReads and
         (__ \ "countryOfResidence").readNullable[String] and
         (__ \ "nationality").readNullable[String] and
-        (__ \ "mentalCapacityYesNo").readNullable[YesNoDontKnow]
+        (__ \ "mentalCapacityYesNo").read[YesNoDontKnow]
```

Comment:
Is there a reason you made this no longer an Option?
It needs to remain an option as the "in progress" logic is determined for only having a name.